Update start pages

alex-ju opened the pull request:
Bring start/landing pages in line with `frontend`. These pages are currently only available in development, as `frontend` takes over and responds to these routes instead. #5277 will change this behaviour, hence the need to align the two apps.

[Trello card](https://trello.com/c/3QJzoCLE)

<table>
<tr><th>Before</th><th>After</th></tr>
<tr><td valign="top">

![localhost_3010_find-coronavirus-support](https://user-images.githubusercontent.com/788096/114075778-d245a880-989d-11eb-8ecb-9bd9f131bf31.png)


</td><td valign="top">

![localhost_3010_find-coronavirus-support](https://user-images.githubusercontent.com/788096/114075261-459aea80-989d-11eb-99fd-425d8fb70d74.png)


</td></tr>
</table>


⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

theseanything commented on app/views/smart_answers/landing.html.erb:
```diff
@@ -1,17 +1,34 @@
+      content_item: @content_item %>
+    <%= render 'govuk_publishing_components/components/machine_readable_metadata',
+      schema: :government_service,
+      content_item: @content_item %>
```
Nit: Its strange, we end up with duplicate twitter meta tags 
```html
<meta name="twitter:card" content="summary">
<meta name="twitter:card" content="summary">
```

Although this pre-exists in Frontend too and probably doesn't cause a problem. But just looked odd.

theseanything commented:
Thanks @alex-ju - looks good! Happy to re-review once the blue banner added.

alex-ju commented:
> Thanks @alex-ju - looks good! Happy to re-review once the blue banner added.

Thank you, @theseanything! I added a commit to replicate [the experiment we run in `frontend`]( https://github.com/alphagov/frontend/pull/2691) on [Find coronavirus financial support for your business](https://smart-answer-update-sta-ly7au7.herokuapp.com/business-coronavirus-support-finder) and rebased to master.
